chart/compass/charts/gateway/templates/deployment.yaml

```diff
@@ -101,6 +108,45 @@ spec:
                   name: {{ .Values.global.auditlog.configMapName }}
                   key: auditlog-oauth-tenant-var
                   optional: true
+            {{ else if eq .Values.gateway.auditlog.authMode "oauth-mtls" }}
+            - name: APP_AUDITLOG_AUTH_MODE
+              value: "oauth-mtls"
+            - name: APP_AUDITLOG_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientIdKey }}
+            - name: APP_AUDITLOG_SKIP_SSL_VALIDATION
+              value: "{{ .Values.global.auditlog.skipSSLValidation }}"
+            - name: APP_AUDITLOG_X509_CERT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientCertKey }}
+            - name: APP_AUDITLOG_X509_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientKeyKey }}
+            - name: APP_AUDITLOG_OAUTH_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.urlKey }}
+            - name: APP_AUDITLOG_TOKEN_PATH
+              value: {{ .Values.global.auditlog.tokenPath }}
```

Comment:
I guess`global.auditlog.tokenPath` is changed to `/cert/token` for `oauth-mtls` mode but this path is also used for `oauth` mode - https://github.com/kyma-incubator/compass/blob/2f792f99efccff17341bfde2bf50d3ba72618520/chart/compass/charts/gateway/templates/deployment.yaml#L98. This most probably will result in error if someone try to use oauth mode because it will calling /cert/token with wrong configuration. Adjust configuration for both modes